src/Symfony/Component/Notifier/Bridge/Slack/SlackTransport.php

```diff
@@ -69,22 +71,23 @@ protected function doSend(MessageInterface $message): void
         }
 
         $options = $opts ? $opts->toArray() : [];
-        $options['token'] = $this->accessToken;
-        if (!isset($options['channel'])) {
-            $options['channel'] = $message->getRecipientId() ?: $this->chatChannel;
-        }
+
         $options['text'] = $message->getSubject();
-        $response = $this->client->request('POST', 'https://'.$this->getEndpoint().'/api/chat.postMessage', [
-            'body' => array_filter($options),
-        ]);
+        $options['blocks'] = isset($options['blocks']) ? json_decode($options['blocks'], true) : null;
+
+        $response = $this->client->request(
+            'POST',
+            sprintf('%s://%s/%s', 'https', $this->getEndpoint(), $this->path),
+            ['json' => array_filter($options)]
+        );
 
         if (200 !== $response->getStatusCode()) {
-            throw new TransportException(sprintf('Unable to post the Slack message: %s.', $response->getContent(false)), $response);
+            throw new TransportException(sprintf('Unable to post the Slack message: "%s".', $response->getContent(false)), $response);
         }
 
-        $result = $response->toArray(false);
-        if (!$result['ok']) {
-            throw new TransportException(sprintf('Unable to post the Slack message: %s.', $result['error']), $response);
+        $result = $response->getContent(false);
+        if ('ok' !== $result) {
```

Comment:
Is that really their response. just plain "ok" if everything is fine... 
That is strange. =)

Reply:
Indeed... it is really strange. I can confirm that this is their response.
<img width="292" alt="Screen Shot 2020-04-04 at 12 00 04" src="https://user-images.githubusercontent.com/65848/78422994-ec85ee80-766b-11ea-945b-1e68c8f09b25.png">